app/views/site/studentFilterSelection.html.haml

```diff
@@ -54,6 +63,11 @@
         
       
     :javascript
+      $(window).load(function(){
+      $('#cover').fadeOut(2000);
```

Comment:
The schema creation could take more than 2 secs, I would suggest fading out once we are done creating the schema. Alternatively, you can show the gif once the upload button is clicked, I guess we refresh the page once the schema is created (I'm not sure about this), which should remove the gif.